src/hooks.ts

```diff
@@ -41,3 +44,6 @@ export function useTitle(title: string): void {
         };
     }, []);
 }
+
+export const useAppDispatch: () => AppDispatch = useDispatch;
```

Comment:
@mgrabovsky could you please add here some documentation. Or explanation.
If I correctly you are creating a new hook. With a custom name.
Is it really necessary?

And maybe add a type for: `() => AppDispatch`
O remove this type at all from this line.

Reply:
This is simply the approach [recommended by Redux Toolkit](https://redux-toolkit.js.org/tutorials/typescript#define-typed-hooks). I'll add a link to the documentation if it bugs you.